packages/suite/src/components/firmware/DeviceTutorial.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useRef } from 'react';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import TrezorConnect from '@trezor/connect';
+import { Button } from '@trezor/components';
+import { getDeviceModel } from '@trezor/device-utils';
+import {
+    beginOnbordingTutorial,
+    goToNextStep,
+    setDeviceTutorialStatus,
+} from 'src/actions/onboarding/onboardingActions';
+import { useDevice, useDispatch, useSelector } from 'src/hooks/suite';
+import { selectIsActionAbortable } from 'src/reducers/suite/suiteReducer';
+import { selectOnboardingTutorialStatus } from 'src/reducers/onboarding/onboardingReducer';
+import messages from 'src/support/messages';
+import { OnboardingStepBox } from '../onboarding';
+import { Translation } from '../suite';
+import { StyledBackdrop } from '../onboarding/OnboardingStepBox';
+
+const BoxContainer = styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: -60px;
+
+    ${StyledBackdrop} {
+        z-index: 2;
+    }
+
+    .collapsible-card__wrapper {
```

Comment:
nitpick: I'm always trying to avoid styling child component from parent styled component if possible, because it's easier to break it without noticing and here it looks easy to achieve maybe? Just create StyledOnboardingStepBox?

Reply:
84808a186